RFS/src/main/java/com/rfs/ReindexFromSnapshot.java

```diff
@@ -59,18 +61,27 @@ public static class Args {
         @Parameter(names = {"--movement-type"}, description = "What you want to move - everything, metadata, or data.  Default: 'everything'", required = false, converter = MovementType.ArgsConverter.class)
         public MovementType movementType = MovementType.EVERYTHING;
 
+        @Parameter(names = {"--index-template-whitelist"}, description = "List of index template names to migrate (e.g. 'posts_index_template1, posts_index_template2')", required = false)
+        public List<String> indexTemplateWhitelist;
+
+        @Parameter(names = {"--component-template-whitelist"}, description = "List of component template names to migrate (e.g. 'posts_template1, posts_template2')", required = false)
+        public List<String> componentTemplateWhitelist;
```

Comment:
Hmmm, this is actually more potentially confusing than I thought originally.  Per our convo, before ES 7.8, there was only `templates`, so if you use the script to migrate a 6.8 cluster you wouldn't see an appropriate option.

Would you mind changing `--index-template-whitelist` to `--templates-whitelist` and updating the description to indicate it refers to legacy templates for ES 6.8 and index templates for 7.10?  Not a perfect fix long term, but probably enough for the moment.

TBH, this argument list is getting pretty long an gnarly.  Not sure what to do about it right now, but we should think about ways to make it easier to navigate the many different use-cases it's trying to address.

Reply:
Ugh, we'll also want a check so that `--component-templates` aren't supplied to a ES 6.8 migration.

I'm wondering if we want different wrapper scripts for different source versions, or something...

Reply:
Created a follow-up JIRA for this: https://opensearch.atlassian.net/browse/MIGRATIONS-1646

Reply:
Updated as per comment for now. Need to think about this more but having a pattern for handling different arguments for different versions could be nice